packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -1365,3 +1375,129 @@ async function getMarketsSearchResults(
   }
   return Augur.syncableFlexSearch.where(whereObj);
 }
+
+/**
+ * Sets the `lastTradedTimestamp` property for all markets in `marketsResults`.
+ *
+ * @param {DB} db Database object to use for getting `lastTradedTimestamp` info
+ * @param {any[]} marketsResults Array of market objects to add `lastTradedTimestamp` to
+ */
+async function setLastTradedTimestamp(db: DB, marketsResults: any[]): Promise<Array<{}>>  {
```

Comment:
same comment as below:
why any here for marketsResults and not typed?
Also with return value why empty object {} and not typed?